storybook/package.json

```diff
@@ -8,8 +8,10 @@
 		}
 	},
 	"scripts": {
+		"prebuild": "npm run remove-dist",
```

Comment:
Do we really need the `remove-dist` task for storybook? We don't have dev/prod builds in the storybook workspace, so we will always have `dist` folder empty, right?

Reply:
Right now it's still building into the same directory as the dev/prod build of the plugin. We could move the build directory elsewhere such as within the storybook directory then we could remove this. I would do this as a new ticket as there will need to be changes to the GH actions if we want to do this to make sure storybook still publishes correctly.

https://github.com/google/site-kit-wp/blob/c54abcd1c8763cfbf4f5bcf78253cef52d397932/storybook/package.json#L12

Reply:
Ok, let it be it. It doesn't hurt having it here.